pytest_datafiles.py

```diff
@@ -1,88 +1,74 @@
 """
 Module containing a 'datafiles' fixture for pytest Tests.
 """
+import shutil
+from pathlib import Path
+from typing import List
 
 import pytest
-from py import path  # pylint: disable=E0611
 
 from _pytest.config import Config
 
 
 def pytest_configure(config: Config) -> None:
+    """Perform initial configuration."""
     config.addinivalue_line(
         "markers",
-        "datafiles(path, ..., *, keep_top_dir=False, on_duplicate='exception'): "
-        "Paths to copy to tmpdir before the test. 'keep_top_dir': For all "
-        "parameters that represent directories, keep that directory instead of "
-        "only (recursively) copying its content (default is False). Use the "
-        "option 'on_duplicate' to specify the action to take when duplicate "
-        "files/directories are found. Possible values are: exception, ignore and "
-        "replace. The default value is exception.",
+        "datafiles(path, ..., *, keep_top_dir=False, "
+        "on_duplicate='exception'): Paths to copy to tmpdir before the test. "
+        "'keep_top_dir': For all parameters that represent directories, keep "
+        "that directory instead of only (recursively) copying its content "
+        "(default is False). Use the option 'on_duplicate' to specify the "
+        "action to take when duplicate files/directories are found. Possible "
+        "values are: exception, ignore and replace. The default value is "
+        "exception.",
```

Comment:
For future PRs please put such changes into a separate PR. I am all for formatting changes that improve legibility, but they make PRs with functional changes unnecessarily difficult to review because I have to assume every line can contain an important functional change (in case of a docstring a change in parameter type etc.).

Reply:
Agreed. In this case, it was making the linter pass, where the change was not about the linter.